### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by LowlyDBA**
* Update placeholder values in the collection template
* Version set to 0.0.0
* Misc Git/hub settings
* Tweak ansible-test action for our needs
* Add in-solidarity bot

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Will be updated by antsibull-changelog. Do not edit this manually!
```
Comment (briantist) [inline CHANGELOG.rst:1]: This file gets generated by `antsibull-changelog` from changelog fragments. No need to revert the change, just a heads-up that it will get overwritten (assuming we use that release process).

File: README.md [hunk 2]
```diff
@@ -53,82 +47,77 @@ To learn how to maintain / become a maintainer of this collection, refer to the
...
-- [Ansible User guide](https://docs.ansible.com/ansible/devel/user_guide/index.html)
-- [Ansible Developer guide](https://docs.ansible.com/ansible/devel/dev_guide/index.html)
-- [Ansible Collections Checklist](https://github.com/ansible-collections/overview/blob/master/collection_requirements.rst)
-- [Ansible Community code of conduct](https://docs.ansible.com/ansible/devel/community/code_of_conduct.html)
-- [The Bullhorn (the Ansible Contributor newsletter)](https://us19.campaign-archive.com/home/?u=56d874e027110e35dea0e03c1&id=d6635f5420)
-- [Changes impacting Contributors](https://github.com/ansible-collections/overview/issues/45)
+* [Ansible Collection overview](https://github.com/ansible-collections/overview)
+* [Ansible User guide](https://docs.ansible.com/ansible/devel/user_guide/index.html)
+* [Ansible Developer guide](https://docs.ansible.com/ansible/devel/dev_guide/index.html)
+* [Ansible Collections Checklist](https://github.com/ansible-collections/overview/blob/master/collection_requirements.rst)
```
Comment (briantist) [inline README.md:111]: The old URL redirects but the branch has been renamed: https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst

File: README.md [hunk 3]
```diff
@@ -53,82 +47,77 @@ To learn how to maintain / become a maintainer of this collection, refer to the
...
+* [Ansible Community code of conduct](https://docs.ansible.com/ansible/devel/community/code_of_conduct.html)
+* [The Bullhorn (the Ansible Contributor newsletter)](https://us19.campaign-archive.com/home/?u=56d874e027110e35dea0e03c1&id=d6635f5420)
+* [Changes impacting Contributors](https://github.com/ansible-collections/overview/issues/45)
```
Comment (briantist) [inline README.md:114]: Am noticing lots of things that need to be updated in the collection template 😅 this issue has been replaced by a new repo (very recently). No worries we'll get it later.

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -1 +1,4 @@
-releases: {}
+ancestor: null
```
Comment (briantist) [inline changelogs/changelog.yaml:1]: This file will also be overwritten by `antsibull-changelog`, this one might need to be reverted to work correctly, we'll see.

File: codecov.yml [hunk 5]
```diff
@@ -1,3 +1,3 @@
 # CHANGE THIS
 fixes:
-   - "/ansible_collections/NAMESPACE/COLLECTION/::"
+   - "/lowlydba/lowlydba/sqlserver/::"
```
Comment (briantist) [inline codecov.yml:3]: This will need to use the path format as shown in the source, starting with `ansible_collections/`

File: CHANGELOG.rst (hunk 1 above)
Comment (LowlyDBA) [reply CHANGELOG.rst:1]: I think that delete was via antsibull - I did a 0.0.0 release just to test it out, see lines below.
